Compile issue on Mac Sierra (10.12.3)

Hi, probably pilot error but I'm having trouble getting the project to compile.
```
$ ./configure
$ make
g++ -c libtins/benchmark.cpp -g -O2 -std=c++11 -o libtins/benchmark.o
libtins/benchmark.cpp:52:28: error: no member named 'accumulate' in namespace 'std'
    auto total_time = std::accumulate(
                      ~~~~~^
1 error generated.
```
Might be related to this: http://stackoverflow.com/questions/32614922/os-x-clang-wont-use-c11-headers. TBH I'm not sure,  I'm attaching my config.log (which I added a .txt to so I could upload it to github :)

[config.log.txt](https://github.com/mfontanini/packet-capture-benchmarks/files/849493/config.log.txt)

I have a simillar isue, and I get:
libtins/benchmark.cpp: In function ‘int main(int, char**)’:
libtins/benchmark.cpp:52:28: error: ‘accumulate’ is not a member of ‘std’
     auto total_time = std::accumulate<numeric>(
                            ^~~~~~~~~~
libtins/benchmark.cpp:52:39: error: ‘numeric’ was not declared in this scope
     auto total_time = std::accumulate<numeric>(
                                       ^~~~~~~
libtins/benchmark.cpp:52:39: note: suggested alternative: ‘cuserid’
     auto total_time = std::accumulate<numeric>(
                                       ^~~~~~~
                                       cuserid

--- a/libcrafter/benchmark.cpp
+++ b/libcrafter/benchmark.cpp
@@ -2,6 +2,7 @@
 #include <chrono>
 #include <vector>
 #include <algorithm>
+#include <numeric>
 #include <crafter.h>
 #include <pcap.h>
 

--- a/libpcap/benchmark.cpp
+++ b/libpcap/benchmark.cpp
@@ -2,6 +2,7 @@
 #include <chrono>
 #include <vector>
 #include <algorithm>
+#include <numeric>
 #include <pcap.h>
 
 using clock_type = std::chrono::high_resolution_clock;

--- a/libtins/benchmark.cpp
+++ b/libtins/benchmark.cpp
@@ -3,6 +3,7 @@
 #include <chrono>
 #include <string>
 #include <vector>
+#include <numeric>
 
 using namespace Tins;
 

